client/e2e/homepage.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { test, expect } from './fixtures/fixtures';
+import { login, logout } from './utils';
```

Comment:
We re-worked this test [in a recent PR](https://github.com/CDCgov/dibbs-ecr-refiner/blob/main/client/e2e/04%20-%20homepage.spec.ts), so think we can just get rid of this file